db/db_impl/db_impl_files.cc

```diff
@@ -741,7 +741,7 @@ uint64_t PrecomputeMinLogNumberToKeep(
   return min_log_number_to_keep;
 }
 
-Status DBImpl::FinishBestEffortsRecovery() {
+Status DBImpl::FinishRecovery() {
```

Comment:
Not necessarily in this PR, but can we use `SstFileManager` to perform file deletions?

Reply:
can try in another PR, the original code for deleting files is reused.